bps/resources/HSC/DRP-RC2.yaml

```diff
@@ -42,21 +36,21 @@ pipetask:
   healSparsePropertyMaps:
```

Comment:
I see that `templateGen` above requested 5000 and used 5237 of memory; do we want to edit that?